onboarding/main.go

```diff
@@ -122,3 +158,25 @@ func convertRsaPublicKeyAsPemStr(publicKey *rsa.PublicKey) (string, error) {
 
 	return string(publicKeyPem), nil
 }
+
+func getStorageClusterMetadata(ctx context.Context, operatorNamespace string, clientset *kubernetes.Clientset) metav1.PartialObjectMetadata {
+	var storageClusterMetaDataList metav1.PartialObjectMetadataList
+	storageClusterGVKPath := fmt.Sprintf("/apis/%s/%s/namespaces/%s/%s", "ocs.openshift.io", "v1", operatorNamespace, "storageclusters")
```

Comment:
Why are we listing all of the storage clusters? we are part of the ocs operator code base and we should have access to the constant that have the hardcoded name of the relevant storage cluster

Reply:
There is no constant i could find these in the project so i have added these.

Reply:
@rchikatw I don't see any local constant and you still listing all storage clusters. Am I missing something?

Reply:
Do you want me to get a specific storage cluster with name ocs-storagecluster

Reply:
If yes,  I have made the change to get a specific storage cluster with the name(ocs-storagecluster), I have not found this constant in ocs-operator so created a local constant.